kqml/kqml_module.py

```diff
@@ -8,30 +8,32 @@
 from docutils.io import InputError
 
 def translate_argv(raw_args):
-    '''Enables conversion from system arguments.
-    
-    Inputs:
-    ------
-    raw_args -- args taken raw from the system input.
-    
-    Outputs:
+    """Enables conversion from system arguments.
+
+    Parameters
+    ----------
+    raw_args : list
+        Arguments taken raw from the system input.
+
+    Returns
     -------
-    kwargs -- the inputs fromatted as a kwargs dict. To use as input, simply
-            use `KQMLModule(**kwargs)`.
-    '''
+    kwargs : dict
+        The input arguments formatted as a kwargs dict.
+        To use as input, simply use `KQMLModule(**kwargs)`.
+    """
     kwargs = {}
     def get_parameter(param_str):
         for i, a in enumerate(raw_args):
             if a == param_str:
-                assert len(raw_args) == i+2 and raw_args[i+1][0] != '-',\
+                assert len(raw_args) == i+2 and raw_args[i+1][0] != '-', \
                     'All arguments must have a value, e.g. `-testing true`'
                 return raw_args[i+1]
         return None
 
     value = get_parameter('-testing')
     if value is not None and value.lower() in ('true', 't', 'yes'):
             kwargs['testing'] = True
-    
```

Comment:
Your editor leaves spaces in blank lines, try to set it up so that this doesn't happen (or have the editor highlight trailing spaces at the end of lines, which is what I have set up).